# DiscoverCone action
----
# Result
bool discovered_cone
----
# Feedback
string feedback